src/pages/Technician/AddTechnician/technicianSpec.js

```diff
@@ -0,0 +1,18 @@
+import { applySpec, pipe, prop } from 'ramda';
+import moment from 'moment';
+
+const forceBoolean = value => !!value;
+const momentFormat = (value) => moment(value).format('DDMMYYYY');
+
+const TechnicianSpec = {
+  CNH: pipe(prop('dueDateCnh'), momentFormat),
+  external: pipe(prop('external'), forceBoolean),
+  name: prop('name'),
+  plate: pipe(prop('car')),
+  responsibleUser: prop('responsibleUser')
```

Comment:
Aqui podemos usar o pathOr 
```suggestion
  responsibleUser: pathOr('modrp', ['responsibleUser'])
```